packages/nuekit/src/init.js

```diff
@@ -22,7 +23,10 @@ export async function init({ dist, is_dev, esbuild }) {
     return await fs.stat(latest)
 
   } catch {
-    await fs.rmdir(outdir, { recursive: true })
+    try {
+      await fs.rm(outdir, { recursive: true })
+    } catch {}
```

Comment:
Is this try/catch necessary? 
I cherry-pick this commit without the try/catch here, seems no problem here
See https://github.com/nuejs/nue/pull/135#issuecomment-1875652749

Reply:
When the target file does not exist, the program will crash on my end.

Reply:
Makes sense. there are also lots of code like this: https://github.com/search?q=%22fs.rm%22+%22%7D+catch+%7B%7D%22&ref=opensearch&type=code

Reply:
> When the target file does not exist

Also, should we pass a `force: true` to `fs.rm` ?
https://nodejs.org/api/fs.html
> force [<boolean>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#Boolean_type) When true, exceptions will be ignored if path does not exist. Default: false.